packages/preview/simple-preavis/0.1.0/typst.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "simple-preavis"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Mathias APARICIO <https://github.com/mathias-aparicio/>"]
+license = "MIT"
+description = "📖 a french move out letter"
+repository = "https://github.com/mathias-aparicio/simple-preavis"
+categories = ["letter"]
```

Comment:
This is not a valid category, see [this list](https://github.com/typst/packages/blob/main/CATEGORIES.md).